### PR TITLE
Add Python 3 support

### PR DESCRIPTION
**Pull request by 1letter**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,11 +42,17 @@
         "Development Status :: 4 - Beta",
         "Environment :: Web Environment",
         "Framework :: Zope3",
+        "Framework :: Plone",
```
Comment (thomasmassmann) [inline setup.py]: This not a dedicated Plone add-on. We use it for plain zope 3 apps as well.

Comment (thomasmassmann) [reply setup.py]: Ah, we moved this to the collective. But still, Zope would be more appropriate I think.

**Review by thomasmassmann — APPROVED** [review]
Maybe you can adjust the classifiers. But other than that it looks good to me.

File: setup.py (hunk 1 above)
Comment (1letter) [reply setup.py]: Ok, i have removed the Plone classifiers.

**Pull request merged**